script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_8.txt

```diff
@@ -154,20 +154,20 @@ She reverts to her former self.
 "...You monster. To think that you still remain intact after a blow from my lance.
 However―――"
 <Page42>
-Arcueid remains collapsed and does not get up.
-Without moving an inch, Vlov holds up his right hand as if to signal something.
-The rubble around Vlov crumbles.
-Several corpses from beneath the ground crawl out as they groan.
+Arcueid remains collapsed and does not get up.//Accurate but awkward アルクェイドは倒れたまま起き上がらない。
+Without moving from where he stands, Vlov holds up his right hand as if to signal something.//ヴローヴは一歩も動かないまま、何かの合図を送るように右手をかかげた。
+The rubble around Vlov crumbles.//Crumbles sorta reads like the chunks of rubble themselves are becoming smaller, rather than the strewn debris being dislodged / falling down as the corpses rise from beneath it. ヴローヴの周囲の瓦礫が崩れていく。
+Several corpses crawl out from underground, groaning.
 <Page43>
-"This hunt now belongs to me.
-Go forth and pick up the woman's corpse.
-That adaptability. That all-powerful blood surpassing all anguish, in these hands."
+"The spoils of this hunt now belong to me.//この狩りは、もはやおれのものだ
+Go forth and pick up that woman's corpse.
+That adaptability. That all-powerful blood surpassing all anguish, bring it to me."//Surpass feels a bit literal here. Isn't it more like he thinks it will soothe / resolve his anguish? その順応性。あらゆる苦しみを上回る万能の血を、この手に
```

Comment:
transcend maybe. I think soothe is a bit too soft, even if that is the underlying intent.

> That all-powerful blood which transcends all anguish